src/ekp_interface.jl

```diff
@@ -131,4 +131,57 @@ function update_ensemble(
     iter_path = path_to_iteration(output_dir, iteration)
     eki_path = joinpath(iter_path, "eki_file.jld2")
     JLD2.save_object(eki_path, eki)
+    return eki
+end
+
+"""
+    calibrate(experiment_id)
+
+Convenience function for running a full calibration experiment for the given
+`experiment_id`. 
+This function requires the relevant experiment project and model interface to be loaded.
+
+```julia
+import CalibrateAtmos
+
+experiment_id = "surface_fluxes_perfect_model"
+experiment_path = joinpath(pkgdir(CalibrateAtmos), "experiments", experiment_id)
+include(joinpath(experiment_path, "model_interface.jl"))
+include(joinpath(experiment_path, "generate_truth.jl"))
+
+eki = CalibrateAtmos.calibrate(experiment_id)
+```
+"""
+function calibrate(experiment_id)
+    ekp_config =
+        YAML.load_file(joinpath("experiments", experiment_id, "ekp_config.yml"))
+    # initialize the CalibrateAtmos
+    initialize(experiment_id)
+
+    # run experiment with CalibrateAtmos for N_iter iterations
+    N_iter = ekp_config["n_iterations"]
+    N_mem = ekp_config["ensemble_size"]
+    eki = nothing
+    for i in 0:(N_iter - 1)
+        # run G model to produce output from N_mem ensemble members
+        physical_model = get_forward_model(Val(Symbol(experiment_id)))
+
+        for m in 1:N_mem # TODO: parallelize with pmap!
```

Comment:
Is is worth adding the pmap here or shall we not bother for now?

Reply:
I tried to add ClimaComms multithreading quickly, but I ran into issues with multiple threads reading from JLD2, so I think I will leave it for another PR.

Reply:
Actually, I went back and added locking around the JLD2 reading, so we now have basic multithreading!